src/analysis/retail/monk/mistweaver/modules/features/MistyPeaksHealingBreakdown.tsx

```diff
@@ -0,0 +1,91 @@
+import { formatThousands } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { TALENTS_MONK } from 'common/TALENTS';
+import { SpellLink } from 'interface';
+import Analyzer, { Options } from 'parser/core/Analyzer';
+import DonutChart from 'parser/ui/DonutChart';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { SPELL_COLORS } from '../../constants';
+import DancingMists from '../spells/DancingMists';
+import MistsOfLife from '../spells/MistsOfLife';
+import MistyPeaks from '../spells/MistyPeaks';
+import RapidDiffusion from '../spells/RapidDiffusion';
+
+class MistyPeaksHealingBreakdown extends Analyzer {
+  get remainingMistyPeaksHealing() {
+    return (
+      this.mistyPeaks.extraHealing -
+      this.rapidDiffusion.mistyPeaksHealingFromRapidDiffusion -
+      this.dancingMists.mistyPeaksHealingFromDancingMist
+    );
+  }
+  static dependencies = {
+    mistyPeaks: MistyPeaks,
+    dancingMists: DancingMists,
+    rapidDiffusion: RapidDiffusion,
+    mistsOfLife: MistsOfLife,
+  };
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS_MONK.MISTY_PEAKS_TALENT.id);
+    if (!this.active) {
+      return;
+    }
+  }
+
+  protected mistyPeaks!: MistyPeaks;
+  protected dancingMists!: DancingMists;
+  protected rapidDiffusion!: RapidDiffusion;
+  protected mistsOfLife!: MistsOfLife;
+
+  renderMistyPeaksChart() {
+    const items = [
+      {
+        color: SPELL_COLORS.ENVELOPING_MIST,
+        label: 'Dancing Mists',
+        spellId: TALENTS_MONK.DANCING_MISTS_TALENT.id,
+        value: this.dancingMists.mistyPeaksHealingFromDancingMist,
+        valueTooltip: formatThousands(this.dancingMists.mistyPeaksHealingFromDancingMist),
+      },
+      {
+        color: SPELL_COLORS.RAPID_DIFFUSION,
+        label: 'Rapid Diffusion',
+        spellId: TALENTS_MONK.RAPID_DIFFUSION_TALENT.id,
+        value: this.rapidDiffusion.mistyPeaksHealingFromRapidDiffusion,
+        valueTooltip: formatThousands(this.rapidDiffusion.mistyPeaksHealingFromRapidDiffusion),
+      },
+      {
+        color: SPELL_COLORS.RENEWING_MIST,
+        label: 'Renewing Mist',
+        spellId: SPELLS.RENEWING_MIST_HEAL.id,
+        value: this.remainingMistyPeaksHealing,
+        valueTooltip: formatThousands(this.remainingMistyPeaksHealing),
+      },
+      // {
+      //   color: SPELL_COLORS.RENEWING_MIST ,
+      //   label: 'Mists of Life',
+      //   spellId: TALENTS_MONK.MISTS_OF_LIFE_TALENT.id,
+      //   value: this.mistsOfLife,
+      //   valueTooltip: formatThousands(this.mistsOfLife),
+      // },
```

Comment:
Uncomment this in this PR as the other one should get landed first (easier to review it for maintainers)

Reply:
The other PR has the refresh event handling for that, waiting to call that property that is implemented in there. Until then there is nothing to add to this chart